network-config-analyzer/NetworkConfig.py

```diff
@@ -163,33 +152,44 @@ def clone_with_just_one_policy(self, name_of_policy_to_include, policy_type=None
         res.append_policy_to_config(policy)
         return res
 
-    def get_captured_pods(self):
+    def get_captured_pods(self, layer_name=None):
         """
-        :return: All pods captured by any policy
+        :param NetworkLayerName layer_name: The name of a layer to get the pods from
+        :return: All pods captured by any policy, in at least one layer
         :rtype: Peer.PeerSet
         """
+        # TODO: should ignore Ingress layer?
```

Comment:
I think Ingress should NOT be ignored. We may miss connection restriction this way.

Reply:
done